tests/net/dhcpv6/src/main.c

```diff
@@ -0,0 +1,1267 @@
+/*
```

Comment:
Thanks for writing tests :-)

Just a suggestion - not blocking by any means - but it might be helpful to separate different aspects of testing into different files. Keep the `main.c` for general setup-before-after, teardown and state.

It might be easier to know where to put additional tests in the future.